packages/modules/create-imodel/src/components/create-imodel/CreateIModel.tsx

```diff
@@ -116,10 +116,11 @@ export function CreateIModel(props: CreateIModelProps) {
         headers: {
           Authorization: `${accessToken}`,
           Prefer: "return=representation",
+          Accept: "application/vnd.bentley.itwin-platform.v2+json",
           "Content-Type": "application/json",
         },
         body: JSON.stringify({
```

Comment:
hmm... Do you have plans to migrate these requests to just use the ts client?

Reply:
No plans on my side

Reply:
@veekeys Which ts client are you speakin of ?

Reply:
https://github.com/iTwin/imodels-clients/tree/main/clients/imodels-client-management

Reply:
Filed #81 so we dont loose sight on this.